app/css/dialog.scss

```diff
@@ -0,0 +1,74 @@
+// These styles are only use for the privacy modal. This is meant to look like sscom.
```

Comment:
This file is being built manually. It's the only scss files so I fugred there wasn't much point having an entire build system just for it.